style.css

```diff
@@ -564,3 +564,528 @@ button {
+  }
+
+  .card-prg-5 {
+    width: 274px;
+    height: 64px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 13px;
+    line-height: 21px;
+    color: #fff;
+    margin: 276px -326px;
+    padding: 0;
+  }
+
+  .desktop-card6 {
+    display: flex;
+    background-image: url(/images/desk-4.png);
+    background-repeat: no-repeat;
+    background-size: 347px;
+    width: 297px;
+    height: 414px;
+    margin-left: 15px;
+  }
+
+  .Profesional-sixth {
+    width: 318px;
+    font-family: 'Crete Round', sans-serif;
+    font-weight: 400;
+    font-size: 30px;
+    line-height: 44px;
+    color: #fff;
+    margin: 212px 12px;
+  }
+
+  .card-prg-6 {
+    width: 274px;
+    height: 64px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 13px;
+    line-height: 21px;
+    color: #fff;
+    margin: 276px -326px;
+    padding: 0;
+  }
+
+  .card-1 {
+    display: none;
+  }
+
+  .btn-2 {
+    width: 268px;
+  }
+
+  .about-section {
+    display: grid;
+  }
+
+  .container-2 {
+    display: flex;
+    background-color: #1c1a19;
+    margin-top: 120px;
+    height: 59%;
+    padding-left: 24px;
+    background-image: url(./images/illustration-about\ me@2x.png);
+    background-repeat: no-repeat;
+    background-size: 515px;
+    background-position: right bottom;
+    width: 100%;
+  }
+
+  .about-l {
+    width: 566px;
+    font-family: 'Crete Round', sans-serif;
+    font-weight: 400;
+    font-size: 99px;
+    line-height: 82px;
+    padding-left: 40px;
+  }
+
+  .prgs {
+    width: 566px;
+    font-weight: 400;
+    font-size: 30px;
+    line-height: 32px;
+    padding-top: 60px;
+    margin: 32px 36px;
+  }
+
+  .see-proj {
+    margin: 129px 45px;
+  }
+
+  .hr {
+    margin: 122px auto;
+    width: 80%;
+    margin-top: -2px;
+  }
+
+  .skills {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .skills-display {
+    display: flex;
+    gap: 150px;
+  }
+
+  .lang-skill {
+    display: flex;
+    flex-direction: column;
+  }
+
+  li.main {
+    width: 228px;
+    height: 24px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  li.li {
+    height: 62px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    display: flex;
+    align-items: center;
+  }
+
+  .form-section {
+    margin: auto;
+  }
+
+  .form {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    width: fit-content;
+    height: inherit;
+    text-align: center;
+    margin-top: -279px;
+    margin-left: 184px;
+  }
+
+  article {
+    width: 488px;
+    height: 261px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+    font-size: 38px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #262626;
+    margin-top: 86px;
+    margin-left: -726px;
+  }
```

Comment:
- [ ] The form section is also not responsive and the text where you tell visitors that you are interested in hearing what they have to say is cut out on some view screens. This is because the section is not responsive. Kindly make the page responsive and fix the error. 👍 

![chrome-capture (2)](https://user-images.githubusercontent.com/77106962/160725351-85110085-fb43-4414-b514-1b4268e5f5a7.gif)